src/olympia/devhub/templates/devhub/versions/list.html

```diff
@@ -128,7 +138,7 @@
             <th>{{ _('Currently on AMO') }}</th>
             <th>{{ _('Status') }}</th>
             <th>{{ _('Validation') }}</th>
-            <th class="version-delete">{{ _('Delete') }}</th>
+            <th class="version-delete">{{ _('Delete/Disable') }}</th>
```

Comment:
Note I'm not changing the functionality of this button or column or dialog in any way - I just changed the header text slightly (with the aim of) making it clearer what it does.